src/base/Syntax.ml

```diff
@@ -178,7 +180,15 @@ let parse_builtin s loc =
   | "to_uint64" -> Builtin_to_uint64
   | "to_uint128" -> Builtin_to_uint128
   | "to_nat" -> Builtin_to_nat
-  | _ -> raise (SyntaxError (sprintf "\"%s\" is not a builtin" s, loc))
+  | _ -> (
+      let err = SyntaxError (sprintf "\"%s\" is not a builtin" s, loc) in
```

Comment:
Btw, if we care for speed, we should not create an error prematurely, only when it's not really needed.

Reply:
This doesn't seem like that expensive an object to create right? Compared to regex for example. I put it at the beginning because it has two uses.

Reply:
I think I'll remove that comment as the code we have now seems short enough (compared to longer thing I had before)